radio/src/storage/storage_common.cpp

```diff
@@ -132,33 +130,52 @@ void checkExternalAntenna()
 {
   if (isModuleXJT(INTERNAL_MODULE)) {
     if (g_eeGeneral.antennaMode == ANTENNA_MODE_EXTERNAL) {
+      // TRACE("checkExternalAntenna(): External");
       globalData.externalAntennaEnabled = true;
-    }
-    else if (g_eeGeneral.antennaMode == ANTENNA_MODE_PER_MODEL && g_model.moduleData[INTERNAL_MODULE].pxx.antennaMode == ANTENNA_MODE_EXTERNAL) {
+    } else if (g_eeGeneral.antennaMode == ANTENNA_MODE_PER_MODEL &&
```

Comment:
I'd like to suggest moving this code to PXX1 specific implementation file, so that it does not clutter the generic parts in `storage_common.cpp`.